glide.yaml

```diff
@@ -1,7 +1,7 @@
 package: github.com/kubernetes-incubator/external-dns
 import:
 - package: github.com/sirupsen/logrus
-  version: ~0.11.5
+  version: ~1.0.3
```

Comment:
@ideahitme this is the fix. logrus@0.11.5 references itself via "github.com/Sirupsen/logrus" internally (e.g. https://github.com/sirupsen/logrus/blob/v0.11.5/hooks/syslog/syslog.go#L7), which doesn't work anymore since the repo has been renamed back to all-lowercase. All internal references are fixed to use lowercase since logrus@1.0.0 so upgrading fixed the issue.

Reply:
the thing is this didn't work for me locally, which could be due to some glide caching. The issue I was facing is related to what @linki showed above, which seems to only occur on Mac OS X.

Reply:
Don't think glide matters since all vendor dependencies are included in this repo. You shouldn't even need glide unless you are installing/upgrading dependencies.

If you get a clean checkout of this branch, you should be able to build it straightaway.

What error are seeing? (I'm also on Mac OS)